README.md

```diff
@@ -110,7 +113,8 @@ $ git pull
 *   curl - Used to download files
 *   sed, grep, cat, tar, sort, tr, uname - General Unix command line utilities
 *   patch - For patching versions that need patching
-*   make -  Builds PostgreSQL
+*   make - Builds PostgreSQL
+*   git - Used to download the current source tree
```

Comment:
Should be dev, yes?

Reply:
`git` as the program is the dependency, `dev` is the command name, or are you referrring to the source tree? I change it to explain better that git is used to get the development source tree.